image_classification/mobilenet_nhwc.js

```diff
@@ -40,6 +40,8 @@ export class MobileNetV2Nhwc {
       } else {
         options.activation = this.builder_.clamp({minValue: 0, maxValue: 6});
       }
+    } else {
+      options.activation = undefined;
```

Comment:
Should we avoid reusing options object across ops? There might be other members we should handle. We can create a temporary object and pass it, like we did in other places, 
```js
const conv1x1Relu6 = await this.buildConv_(
        input, weightsNameArray[0], `${biasPrefix}_expand_Conv2D`, true,
        {autoPad, filterLayout: 'ohwi'});
```

Reply:
Right, that makes sense! 

Fixed and reverted unnecessary changes.